### PR TITLE
plasma-infra: Enable "New hope" storybook

### PR DESCRIPTION
**Pull request by Yakutoc**
### What/why Changed

Включен storybook для пакета `plasma-new-hope`. 

------

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.790.6404021234.ts'>borderRadius_default_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.790.6404021234.ts'>borderRadius_plasma_b2c_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.790.6404021234.ts'>borderRadius_plasma_web_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.790.6404021234.ts'>borderRadius_sberHealth_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.790.6404021234.ts'>borderRadius_sbermarket_business_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.790.6404021234.ts'>borderRadius_sbermarket_metro_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.790.6404021234.ts'>borderRadius_sbermarket_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.790.6404021234.ts'>borderRadius_sbermarket_selgros_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.790.6404021234.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.790.6404021234.ts'>borderRadius_sberonline_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.790.6404021234.ts'>borderRadius_sberprime_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.790.6404021234.ts'>borderRadius_stylesSalute_react-native--canary.790.6404021234.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.790.6404021234.swift'>color_default_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.790.6404021234.kt'>color_default_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.790.6404021234.ts'>color_default_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.790.6404021234.xml'>color_default_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.790.6404021234.swift'>color_plasma_b2c_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.790.6404021234.kt'>color_plasma_b2c_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.790.6404021234.ts'>color_plasma_b2c_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.790.6404021234.xml'>color_plasma_b2c_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.790.6404021234.swift'>color_plasma_web_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.790.6404021234.kt'>color_plasma_web_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.790.6404021234.ts'>color_plasma_web_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.790.6404021234.xml'>color_plasma_web_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.790.6404021234.swift'>color_sberHealth_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.790.6404021234.kt'>color_sberHealth_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.790.6404021234.ts'>color_sberHealth_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.790.6404021234.xml'>color_sberHealth_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.790.6404021234.swift'>color_sbermarket_business_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.790.6404021234.kt'>color_sbermarket_business_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.790.6404021234.ts'>color_sbermarket_business_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.790.6404021234.xml'>color_sbermarket_business_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.790.6404021234.swift'>color_sbermarket_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.790.6404021234.kt'>color_sbermarket_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.790.6404021234.swift'>color_sbermarket_metro_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.790.6404021234.kt'>color_sbermarket_metro_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.790.6404021234.ts'>color_sbermarket_metro_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.790.6404021234.xml'>color_sbermarket_metro_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.790.6404021234.ts'>color_sbermarket_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.790.6404021234.swift'>color_sbermarket_selgros_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.790.6404021234.kt'>color_sbermarket_selgros_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.790.6404021234.ts'>color_sbermarket_selgros_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.790.6404021234.xml'>color_sbermarket_selgros_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.790.6404021234.swift'>color_sbermarket_wlbusiness_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.790.6404021234.kt'>color_sbermarket_wlbusiness_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.790.6404021234.ts'>color_sbermarket_wlbusiness_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.790.6404021234.xml'>color_sbermarket_wlbusiness_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.790.6404021234.xml'>color_sbermarket_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.790.6404021234.swift'>color_sberonline_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.790.6404021234.kt'>color_sberonline_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.790.6404021234.ts'>color_sberonline_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.790.6404021234.xml'>color_sberonline_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.790.6404021234.swift'>color_sberprime_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.790.6404021234.kt'>color_sberprime_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.790.6404021234.ts'>color_sberprime_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.790.6404021234.xml'>color_sberprime_xml--canary.790.6404021234.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.790.6404021234.swift'>color_stylesSalute_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.790.6404021234.kt'>color_stylesSalute_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.790.6404021234.ts'>color_stylesSalute_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.790.6404021234.xml'>color_stylesSalute_xml--canary.790.6404021234.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.790.6404021234.ts'>shadow_default_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.790.6404021234.ts'>shadow_plasma_b2c_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.790.6404021234.ts'>shadow_plasma_web_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.790.6404021234.ts'>shadow_sberHealth_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.790.6404021234.ts'>shadow_sbermarket_business_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.790.6404021234.ts'>shadow_sbermarket_metro_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.790.6404021234.ts'>shadow_sbermarket_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.790.6404021234.ts'>shadow_sbermarket_selgros_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.790.6404021234.ts'>shadow_sbermarket_wlbusiness_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.790.6404021234.ts'>shadow_sberonline_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.790.6404021234.ts'>shadow_sberprime_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.790.6404021234.ts'>shadow_stylesSalute_react-native--canary.790.6404021234.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.790.6404021234.ts'>spacing_default_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.790.6404021234.ts'>spacing_plasma_b2c_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.790.6404021234.ts'>spacing_plasma_web_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.790.6404021234.ts'>spacing_sberHealth_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.790.6404021234.ts'>spacing_sbermarket_business_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.790.6404021234.ts'>spacing_sbermarket_metro_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.790.6404021234.ts'>spacing_sbermarket_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.790.6404021234.ts'>spacing_sbermarket_selgros_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.790.6404021234.ts'>spacing_sbermarket_wlbusiness_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.790.6404021234.ts'>spacing_sberonline_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.790.6404021234.ts'>spacing_sberprime_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.790.6404021234.ts'>spacing_stylesSalute_react-native--canary.790.6404021234.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.790.6404021234.swift'>typo_mage_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.790.6404021234.kt'>typo_mage_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.790.6404021234.ts'>typo_mage_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.790.6404021234.swift'>typo_plasma_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.790.6404021234.kt'>typo_plasma_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.790.6404021234.ts'>typo_plasma_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.790.6404021234.swift'>typo_ruler_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.790.6404021234.kt'>typo_ruler_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.790.6404021234.ts'>typo_ruler_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.790.6404021234.swift'>typo_sage_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.790.6404021234.kt'>typo_sage_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.790.6404021234.ts'>typo_sage_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.790.6404021234.swift'>typo_sbermarket_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.790.6404021234.kt'>typo_sbermarket_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.790.6404021234.ts'>typo_sbermarket_react-native--canary.790.6404021234.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.790.6404021234.swift'>typo_soulmate_ios-swift--canary.790.6404021234.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.790.6404021234.kt'>typo_soulmate_kotlin--canary.790.6404021234.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.790.6404021234.ts'>typo_soulmate_react-native--canary.790.6404021234.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.236.3-canary.790.6404021234.0
  npm install @salutejs/plasma-new-hope@0.5.3-canary.790.6404021234.0
  npm install @salutejs/plasma-web@1.236.3-canary.790.6404021234.0
  # or 
  yarn add @salutejs/plasma-b2c@1.236.3-canary.790.6404021234.0
  yarn add @salutejs/plasma-new-hope@0.5.3-canary.790.6404021234.0
  yarn add @salutejs/plasma-web@1.236.3-canary.790.6404021234.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-790/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-790/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-790/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-790/web-storybook/
new-hope storybook: http://plasma.sberdevices.ru/pr/pr-790/new-hope-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-790/

File: .github/workflows/documentation-main.yml [hunk 1]
```diff
@@ -79,6 +80,11 @@ jobs:
           npm run storybook:build --prefix="./packages/plasma-temple"
           cp -R ./packages/plasma-temple/build-sb ./s3_build/next-${{ github.sha }}/temple-storybook
 
+      - name: Plasma "New Hope" Storybook
```
Comment (Yakutoc) [inline .github/workflows/documentation-main.yml:83]: [Уточнение]:

Нам нужен stotybook в main документации?

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-790/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-790/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-790/web-storybook/
new-hope storybook: http://plasma.sberdevices.ru/pr/pr-790/new-hope-storybook/

File: .github/workflows/documentation-main.yml (hunk 1 above)
Comment (neretin-trike) [reply .github/workflows/documentation-main.yml:83]: кажется нет 🤔

Comment (Yakutoc) [reply .github/workflows/documentation-main.yml:83]: Я же правильно понимаю, что это нужно для разработки/показать? 

Если да, то тогда действительно не нужно в мастере

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-790/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-790/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-790/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-790/web-storybook/
new-hope storybook: http://plasma.sberdevices.ru/pr/pr-790/new-hope-storybook/

File: .github/processing-scope.js [hunk 2]
```diff
@@ -40,8 +40,9 @@ module.exports = () => {
     const HAS_PLASMA_WEB = packagesList.includes('@salutejs/plasma-web');
     const HAS_PLASMA_B2C = packagesList.includes('@salutejs/plasma-b2c');
     const HAS_PLASMA_TEMPLE = packagesList.includes('@salutejs/plasma-temple');
+    const HAS_PLASMA_HOPE = packagesList.includes('@salutejs/plasma-new-hope');
 
-    const HAS_DOCUMENTATION_CHANGED = HAS_PLASMA_UI_DOCS || HAS_PLASMA_WEB_DOCS || HAS_PLASMA_TEMPLE_DOCS;
+    const HAS_DOCUMENTATION_CHANGED = [HAS_PLASMA_UI_DOCS, HAS_PLASMA_WEB_DOCS, HAS_PLASMA_TEMPLE_DOCS].includes(true);
```
Comment (neretin-trike) [inline .github/processing-scope.js:45]: я немного сломался тут. Мб лучше? Оно и выполнится чуть быстрее, т.к. при первой встрече с true прекратит выполнение функции

```
[HAS_PLASMA_UI_DOCS, HAS_PLASMA_WEB_DOCS, HAS_PLASMA_TEMPLE_DOCS].some(Boolean);
```

Comment (Yeti-or) [reply .github/processing-scope.js:45]: мне кажется на трех элементах оно выполнится одинаково

File: .github/workflows/documentation-main.yml [hunk 3]
```diff
@@ -16,13 +16,14 @@ jobs:
     steps:
       # [NOTE]: В проекте default branch - dev, что бы правильно собрать
       # актуальную версию для branch master, нужно указать ref = 'master'
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```
Comment (Yeti-or) [inline .github/workflows/documentation-main.yml:19]: это ничего не поменяет?

**Review by Yeti-or — APPROVED** [review]

File: .github/workflows/documentation-main.yml (hunk 3 above)
Comment (Yakutoc) [reply .github/workflows/documentation-main.yml:19]: @Yeti-or 

В checkout@v4 добавили

```
Updated default runtime to node20
This requires a minimum Actions Runner version of [v2.308.0](https://github.com/actions/runner/releases/tag/v2.308.0).
Added support for fetching without the --progress option
```
 
Я вот из-за этого перешел 

> Added support for fetching without the --progress option

File: .github/processing-scope.js (hunk 2 above)
Comment (Yakutoc) [reply .github/processing-scope.js:45]: Я тут подумал, что скорее всего старый вариант верну. Увеличение переменных не предвидеться.

Comment (neretin-trike) [reply .github/processing-scope.js:45]: > мне кажется на трех элементах оно выполнится одинаково

на трёх да, но у меня уже привычка думать о расширении

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-790/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-790/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-790/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-790/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-790/web-storybook/
new-hope storybook: http://plasma.sberdevices.ru/pr/pr-790/new-hope-storybook/

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.236.3-dev.0, @salutejs/plasma-new-hope@0.5.4-dev.0, @salutejs/plasma-temple-docs@0.173.4-dev.0, @salutejs/plasma-ui-docs@0.222.4-dev.0, @salutejs/plasma-web-docs@0.190.3-dev.0, @salutejs/plasma-web@1.236.3-dev.0, @salutejs/plasma-website@0.198.3-dev.0 🚀

**Pull request merged**